web_server/comments/migrations/0002_comment_content.py

```diff
@@ -1,19 +0,0 @@
-# Generated by Django 3.0.2 on 2020-02-01 04:31
```

Comment:
Why are you deleting migration files? These need to stay around so that everyone can migrate their existing DB to the schema you have defined.

Reply:
I regenerated new migration file for Comment, cuz previously Comment is linking old Author model and I changed it so that it references the new Author model.

Reply:
I see, ok then, not really sure how to get around this then -> that means this is a breaking change. If I understand migrations correctly if someone doesn't have the latest set of migrations and then attempts to migrate they might not be able to - will have to keep an eye out after we merge if someone says they get an error when migrating.